content/_events/2020/april.md

```diff
@@ -0,0 +1,126 @@
+---
+published: false
+upcoming: true
+title: Documentation templates for fun and profit
+event-id: april-2020-documentation-templates
+year: 2020
+event-date: Thursday, April 23, 2020
+locale: san-francisco
+location: google-345-spear-street
+rsvp_open: true
```

Comment:
We should set this to false until we have a meetup link or the form....having some problems with this include anyway.